Dockerfile.ci

```diff
@@ -138,6 +138,7 @@ RUN mkdir -pv /usr/share/man/man1 \
       # To run virtualenv tests with python 2
       libpython2.7-stdlib \
       lsb-release \
+      nano \
```

Comment:
Can you please add it much further than here (even in a new separate apt-get command - closer to where cloud and kubectl is installed? It belongs there and we will be able to use it later to add more tools like that. There is virtually no benefit in having it here and by modifying this command you cause the image to invalidate 1.5 GB cache so you forces everyone who has breeze to wait 10 minutes just to install nano.

Reply:
See here: https://github.com/PolideaInternal/airflow/pull/937/checks?check_run_id=619914451#step:5:1 - the build takes 18 minutes instead of 4m39 seconds.

Reply:
I added it here because we are installing vim in this block. I will move it.

Reply:
Yep I know. But vi is ESSENTIAL ;)